.github/workflows/release-please.yml

```diff
@@ -16,3 +16,35 @@ jobs:
           command: manifest
           token: ${{secrets.GITHUB_TOKEN}}
           default-branch: main
+    outputs:
+      release_created: ${{ steps.release.outputs.release_created }}
+      release_tag_name: ${{ steps.release.outputs.tag_name }}
+
+  release-assets:
+    needs: release-please
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+          with:
+            submodules: recursive
```

Comment:
I'm not sure if we need the submodule anymore. Also, you may want to checkout the tagged version like we did in the Go SDK.

https://github.com/open-feature/go-sdk/blob/main/.github/workflows/release-please.yml#L35

Reply:
done